### DISCUSSION TITLE
"ReferenceError: $ is not defined" in my pc browser

### DISCUSSION THREAD

**Discussion by JinOuYongGu** [plan-player-analytics/Plan#2396] [Archive]
This didn't happen when I used VPN:

On my PC Chrome browser the webpage shows

`Error occurred, see Developer Console (Ctrl+Shift+I) - Please report this: ReferenceError: $ is not defined`

And I found some log in dev console below:

`GET https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css net::ERR_CONNECTION_TIMED_OUT
network:1046          GET https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js net::ERR_CONNECTION_TIMED_OUT
network:1049          GET https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js net::ERR_CONNECTION_TIMED_OUT
datatables.min.js?v=5.4%20build%201690:33 Uncaught ReferenceError: jQuery is not defined
    at datatables.min.js?v=5.4%20build%201690:33:284
    at datatables.min.js?v=5.4%20build%201690:33:309
(anonymous) @ datatables.min.js?v=5.4%20build%201690:33
(anonymous) @ datatables.min.js?v=5.4%20build%201690:33
sb-admin-2.js?v=5.4%20build%201690:62 Uncaught ReferenceError: $ is not defined
    at sb-admin-2.js?v=5.4%20build%201690:62:1
(anonymous) @ sb-admin-2.js?v=5.4%20build%201690:62
color-selector.js?v=5.4%20build%201690:498 Uncaught ReferenceError: jQuery is not defined
    at color-selector.js?v=5.4%20build%201690:498:4
(anonymous) @ color-selector.js?v=5.4%20build%201690:498
network:1139 Uncaught ReferenceError: $ is not defined
    at network:1139:9
(anonymous) @ network:1139
DevTools failed to load source map: Could not load content for http://plan.mclixiang.com/vendor/highcharts/modules/drilldown.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://plan.mclixiang.com/vendor/highcharts/modules/map.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://plan.mclixiang.com/vendor/highcharts/modules/no-data-to-display.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://plan.mclixiang.com/vendor/highcharts/highstock.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
network:1155 Uncaught ReferenceError: $ is not defined
    at network:1155:13`

**Comment by JinOuYongGu** [comment]
I'm not sure if it's because of the browser. My phone and my computer are on the same wifi, but my phone is fine

**Comment by AuroraLS3** [comment]
Looks like your browser failed to connect to https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css and https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js and https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js

This may be due to your local DNS.
You can replace the URLs to jsdelivr.net in .html files with https://github.com/plan-player-analytics/Plan/wiki/Html-Customization

**Comment by JinOuYongGu** [comment]
Thanks for your timely support! I am sure the problem is due to DNS.
I replaced the cdn.jsdelivr.net into fastly.jsdelivr.net
And my plan looks fine
